src/asm/lexer.c

```diff
@@ -679,6 +682,8 @@ static void freeExpansion(struct Expansion *expansion)
 		child = next;
 	}
 	free(expansion->name);
+	if (expansion->owned)
+		free((char *)expansion->contents);
```

Comment:
[GCC doesn't like this, apparently](https://github.com/gbdev/rgbds/runs/1543959054?check_suite_focus=true#step:4:65). It should be fine to remove `const` from the struct, then.

Reply:
I was about to post a question about that. Removing the `const` from the struct only pushes the problem back, since various `const char *`s get assigned to `contents`. A lot of `const` qualifiers would need removing to avoid this warning.

Reply:
I think that's been solved by storing a `char *owned` instead of a `bool owned`, with it being `NULL` instead of `false` for typical un-owned const expansions.

Reply:
Now storing a `union` of `char *` and `const char *` instead. It's still a hacky workaround; `#pragma GCC diagnostic push/ignored/pop` might be preferable (it'd still be verbose but would keep the extra typing around the line where it's relevant).